README.md

```diff
@@ -1,56 +1,48 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Book Store
 
-# Project Name
-
-> Description the project.
+## Screenshot ![see](./screenshot.png?raw=true 'Title')
 
+> The app retrieves available tickers from the API lists them in an home page. When a particular ticker is clicker it routes to the information of that ticker.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+- React
 
```

Comment:
- [ ] When I click on your live demo link it gives me an error. Please try and fix that.